app/models/dog_searcher.rb

```diff
@@ -31,25 +33,24 @@ def initialize(params: {}, manager: false)
   end
 
   def search
-    @dogs = Dog.includes(:photos, :foster)
-    if @manager
-      @dogs = @dogs.includes(:adopters, :comments)
+    if @manager # manager view
+      @dogs = Dog.includes(:adoptions, :adopters, :comments)
```

Comment:
FYI this class is getting a major refactoring when the filter-by-breed feature is added. I stopped cleaning up here, for Rails 5.2 and deferred it to the next feature I'm working on (filter-by-breed).